tests/snapshots/solver__merge_installable_non_continuous_range.snap

```diff
@@ -0,0 +1,10 @@
+---
+source: tests/solver.rs
+expression: "solve_snapshot(provider, &[\"a 1\", \"a 2..20\"])"
+---
+The following packages are incompatible
+├─ a >=2, <20 can be installed with any of the following options:
+│  └─ a 19 | >=16, <=17 | >=9, <=14 | >=2, <=7
```

Comment:
I was told that `pubgrub` has a implementation of a `simplify` algorithm for ranges like this (it would go and merge all the ranges that overlap so that the result would be the same range but expressed with fewer terms). I think in this case, it would be nice if it was simplified to `>=2,<20` - although that _is_ in fact what the "MatchSpec" above says ... so this thought might be futile anyways.